src/sass/claims-status.scss

```diff
@@ -668,7 +678,21 @@ h1:focus {
   }
 
   .appeal-event-date {
-    color: $color-gray-light;
+    color: $color-gray-medium;
+  }
+
+  & + .down-arrow {
+    width: 0px;
+    border-left: 0.6em solid transparent;
+    border-right: 0.6em solid transparent;
+    border-top: 1.2em solid #d6d7d9;
```

Comment:
Very minor, but is there a variable already for this hex color? Might be `$color-gray-lighter`?

Reply:
😂 
@robbiethegeek Just asked about this too and I made a PR to fix what's already been merged into master. Looped back around to fix this up and saw this comment.

Good call! `$color-gray-lighter` is exactly what I want. 😄